app/src/main/java/org/p2p/wallet/auth/gateway/api/request/RegisterWalletRequest.kt

```diff
@@ -0,0 +1,52 @@
+package org.p2p.wallet.auth.gateway.api.request
+
+import com.google.gson.annotations.SerializedName
+import java.util.UUID
+
+data class RegisterWalletRequest(
+    @SerializedName("params")
+    val params: Params
+) {
+    class Params(
+        /**
+         * Solana pubkey base58 encoded
+         */
+        @SerializedName("client_id")
+        val clientPublicKeyB58: String,
+        @SerializedName("ethereum_id")
+        val etheriumPublicKeyB58: String,
```

Comment:
etherium address is a hex of the first 20 bytes of etherium pubkey, not a base58 (like solana pubkey)